src/System.Net.Security/src/System/Net/Windows/CertModule.cs

```diff
@@ -0,0 +1,180 @@
+                    cppStruct.dwFlags |= (Interop.Crypt32.CertChainPolicyIgnoreFlags.CERT_CHAIN_POLICY_IGNORE_ALL &
+                                        ~Interop.Crypt32.CertChainPolicyIgnoreFlags.CERT_CHAIN_POLICY_IGNORE_INVALID_NAME_FLAG);
+
+                    SafeX509ChainHandle chainContext = chain.SafeHandle;
+
+                    status = Verify(chainContext, ref cppStruct);
+
+                    if (status == Interop.Crypt32.CertChainPolicyErrors.CERT_E_CN_NO_MATCH)
+                    {
+                        sslPolicyErrors |= SslPolicyErrors.RemoteCertificateNameMismatch;
+                    }
+                }
+            }
+
+            return sslPolicyErrors;
+        }
+
+        //
+        // Extracts a remote certificate upon request.
+        //
+        internal override X509Certificate2 GetRemoteCertificate(SafeDeleteContext securityContext, out X509Certificate2Collection remoteCertificateStore)
+        {
+            remoteCertificateStore = null;
+
+            if (securityContext == null)
+            {
+                return null;
+            }
+
+            GlobalLog.Enter("SecureChannel#" + Logging.HashString(this) + "::RemoteCertificate{get;}");
+            X509Certificate2 result = null;
+            SafeFreeCertContext remoteContext = null;
+            try
+            {
+                int errorCode = SSPIWrapper.QueryContextRemoteCertificate(GlobalSSPI.SSPISecureChannel, securityContext, out remoteContext);
+                if (remoteContext != null && !remoteContext.IsInvalid)
+                {
+                    result = new X509Certificate2(remoteContext.DangerousGetHandle());
+                }
+            }
+            finally
+            {
+                if (remoteContext != null && !remoteContext.IsInvalid)
+                {
+                    remoteCertificateStore = UnmanagedCertificateContext.GetRemoteCertificatesFromStoreContext(remoteContext);
+
+                    remoteContext.Dispose();
+                }
+            }
+
+            if (Logging.On)
+            {
+                Logging.PrintInfo(Logging.Web, SR.Format(SR.net_log_remote_certificate, (result == null ? "null" : result.ToString(true))));
+            }
+
+            GlobalLog.Leave("SecureChannel#" + Logging.HashString(this) + "::RemoteCertificate{get;}", (result == null ? "null" : result.Subject));
+
+            return result;
+        }      
+
+        //
+        // Used only by client SSL code, never returns null.
+        //
+        internal override string[] GetRequestCertificateAuthorities(SafeDeleteContext securityContext)
+        {
+            string[] issuers = Array.Empty<string>();
+
+            object outObj;         
+
+            int errorCode = SSPIWrapper.QueryContextIssuerList(GlobalSSPI.SSPISecureChannel, securityContext, out outObj);
+
+            GlobalLog.Assert(errorCode == 0, "QueryContextIssuerList returned errorCode:"+errorCode);
```

Comment:
Please follow the GlobalLog style within this function. (`SecureChannel::FunctionName():` ) - note that I have forgot to update the GlobalLog myself in the original PR.
Nit: spacing around `+errorCode`.